superset-frontend/src/views/CRUD/alert/types.ts

```diff
@@ -74,7 +75,7 @@ export type AlertObject = {
   owners?: Array<Owner | MetaObject>;
   sql?: string;
   recipients?: Array<Recipient>;
-  report_format?: 'PNG' | 'CSV';
+  report_format?: 'PNG' | 'CSV' | 'TEXT';
```

Comment:
what did you think about calling this 'HTML' instead of  TEXT? cc @yousoph

Reply:
I think for email HTML makes sense, but not for Slack (since it's not technically HTML).

Reply:
Let's stick with "text" then :)